app/build.gradle

```diff
@@ -132,7 +132,7 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:$okhttpVersion"
     implementation "com.github.ligi.snackengage:snackengage-playrate:$snackEngageVersion"
     implementation "de.cketti.mailto:email-intent-builder:$emailIntentBuilderVersion"
-    implementation files("libs/TraceDroid-1.1.jar")
+    implementation "org.ligi:tracedroid:1.4"
```

Comment:
Please define the version along with the other dependency version as `traceDroidVersion = 1.4`. Force push your branch.

Reply:
Done